corehq/warehouse/loaders/synclog.py

```diff
@@ -18,6 +18,7 @@ class SyncLogStagingLoader(HQToWarehouseETLMixin, BaseStagingLoader):
     """
     slug = SYNCLOG_STAGING_SLUG
     model_cls = SyncLogStagingTable
+    log_frequency = 10000
```

Comment:
Are there many fewer synclogs than forms?

Reply:
yes, an order of magnitude less